phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -1222,7 +1228,7 @@ public void createTable(RpcController controller, CreateTableRequest request,
                 // exists without making an additional query
                 PTable table =
                         loadTable(env, tableKey, cacheKey, clientTimeStamp, HConstants.LATEST_TIMESTAMP);
-                if (table != null) {
+                if (table != null && !isTableDeleted(table)) {
```

Comment:
Took a closer look at this change and it doesn't look right. The if statement has a check inside of it for the table not being deleted. Can you revert this change and isolate any tests that are failing?